src/furui/containers/local-timeline/index.tsx

```diff
@@ -10,6 +10,7 @@ export default () => {
   const [uploadState, setUploadState] = useState(0)
   const [files, setFiles] = useState([] as AlbumFile[])
   const [draftDisabled, setDraftDisabled] = useState(false)
+  const [inReplyTo, setInReplyTo] = useState(null as number | null)
```

Comment:
こっちのほうが安全のはず
既にあるコードと同じだしいまのところ目視で問題と取れる範囲でもないので直さなくても大丈夫です

```suggestion
  const [inReplyTo, setInReplyTo] = useState<number | null>(null)
```